back/db/seeds/citizenlab.rb

```diff
@@ -180,6 +180,10 @@ def create_for_current_locales
     public_api_tokens: {
       enabled: true,
       allowed: true
+    },
+    analysis: {
+      enabled: true,
+      allowed: true
```

Comment:
I did forget about all these flags (and in the other 2 files), so please keep those :D